scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -179,6 +181,9 @@ class FossId(
         packageNamespaceFilter = fossIdScannerOptions["packageNamespaceFilter"].orEmpty()
         packageAuthorsFilter = fossIdScannerOptions["packageAuthorsFilter"].orEmpty()
         addAuthenticationToUrl = fossIdScannerOptions["addAuthenticationToUrl"].toBoolean()
+        waitForResult = (fossIdScannerOptions["waitForResult"] ?: "true").toBoolean()
```

Comment:
I believe it's not so nice to hard-code the `true` default as a string here. I propose instead:

    fossIdScannerOptions["waitForResult"]?.toBooleanStrictOrNull() ?: true